news/4461.bugfix

```diff
@@ -0,0 +1,2 @@
+clarifies the error given to suggest to the user
+what might be wrong.
```

Comment:
I didn't mean literally just that text :-( It was just a suggestion on the way to express what had changed.